recipes/conda-export/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "conda-export" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/olegtarasov/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 8635008aea0d8ea542c387d81ebf1b761c59d05f6097a78d882cbdaafea9c909
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - conda-export = conda_export:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - pip
+    - conda
+
+test:
+  commands:
+    - conda-export -h
+
+about:
+  home: https://github.com/olegtarasov/conda-export
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Platform agnostic conda environment export'
+  description: |
+    An alternative to `conda env export` that helps create portable environment 
+    specifications with minimal number of packages. Resulting specification is 
+    similar to `conda env export --from-history`, but also includes packages 
+    that were installed using `pip`.
```

Comment:
@olegtarasov I see you are also the author of the code. Do you know about conda-lock? Is the code you use to figure out pip vs conda install something we could use to improve conda-lock?

Reply:
cc @mariusvniekerk

Reply:
@ocefpaf I didn't know about conda-lock. From what I can gather from the docs, it solves a slightly different problem. The point of conda-export is to create an environment spec file similar to `conda env export --from-history`, adding the packages that were installed separately with pip.

Of course, you are most welcome to use any of my code. But I should warn you that it's pretty naive, probably doesn't handle some tricky edge cases and it uses private `pip` API that can change at any moment. If you find a way to improve it, please ping me so I can incorporate those improvements into conda-export.

Thank you!

Reply:
@ocefpaf I found a better and less hacky way to get the list of pip packages. See here: https://github.com/olegtarasov/conda-export/blob/b363f508d52bd887f925536fba149d70c2fceb10/conda-export.py#L15

Reply:
Nice! Simple and elegant. Thanks!

For conda-lock the main issue is that the env is not "realized" yet. In order to support pip packages there we would need to actually create the envs. While it is possible to do without it I'm not sure it is worth it. However, being able to figure out pip packages in an existing env is already a big win for the community. Thanks for conda-export!!